packages/repository/src/model.ts

```diff
@@ -158,7 +158,7 @@ export abstract class Model {
   /**
    * Convert to a plain object as DTO
    */
-  toObject(options?: Options): Object {
+  toObject(options?: Options): AnyObject {
```

Comment:
Before this change, the result of `toObject` was considered as an object with no properties. The following code is rejected by the compiler:

```ts
modelInstance.toObject().id
```

After this change, the result of `toObject` is considered as object with arbitrary properties, thus the code above passes compiler checks again.